service/frontend/adminHandler_test.go

```diff
@@ -1038,3 +1038,71 @@ func Test_UpdateDomainIsolationGroups(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
Leaving this comment here as a reminder to update documentation https://github.com/uber/Cadence-Docs/pull/181/files